src/accounts.rs

```diff
@@ -1,5 +1,6 @@
 extern crate serde_yaml;
 
+use colored::*;
```

Comment:
Decided on the colored crate over term. Seemed to be a bit easier to control color.